cli/tools/registry/pm/outdated.rs

```diff
@@ -179,6 +180,31 @@ pub async fn outdated(
   let jsr_fetch_resolver =
     Arc::new(JsrFetchResolver::new(file_fetcher.clone()));
 
+  if !cli_options.start_dir.has_deno_json()
+    && !cli_options.start_dir.has_pkg_json()
+  {
+    bail!(
+      "No deno.json or package.json in {:?}.",
+      cli_options.initial_cwd(),
```

Comment:
Nitpick to not rely on debug formatting:

```suggestion
      "No deno.json or package.json in \"{}\".",
      cli_options.initial_cwd().display(),
```